core/plugin.go

```diff
@@ -25,6 +25,7 @@ import (
 	"io/ioutil"
 	"time"
 
+	"github.com/intelsdi-x/snap/control/plugin/cpolicy"
```

Comment:
I realize this is the only way, right now, to make this work, but this dependency appears disorganized to me.

Maybe we can consider a new `interface` type that resides in `core`, which `cpolicy.ConfigPolicy` implements.